libs/shared/ui-components/src/SocialCard/SocialCard.tsx

```diff
@@ -64,17 +73,20 @@ export const SocialCard: FC<TSocialCardProps> = ({
         name="twitter:image"
         content={twitterImage || defaultTwitterImage(variant)}
       />
+      {/**
```

Comment:
Ah, sorry, I wasn't clear.

JSDoc, with the `/**`, is only used for documenting objects/props/etc. If, at some point in the future, we set up a full documentation system, the comments starting with `/**` will tell the documentation tool to treat that comment specially, as documentation for the object it precedes.

For other, general code comments, they should have the standard `/*` opening, to indicate they're not meant to appear in the rendered documentation.

```suggestion
      {/*
```